tensorflow/compiler/xla/service/gpu/instruction_fusion.cc

```diff
@@ -29,12 +29,23 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+bool ElementIsF32OrF16(const Shape& shape) {
```

Comment:
`static` or put under anonymous namespace.

Reply:
My oversight. Thanks for the catch. Will update it soon,